setup.py

```diff
@@ -34,6 +34,7 @@
         'pyyaml',
         'theblues',
         'pyRFC3339>=1.0,<2.0',
+        'macaroonbakery<1.0',
```

Comment:
It would be nice to keep this in alphabetical order.

Reply:
done